near/omni-bridge/src/lib.rs

```diff
@@ -517,11 +547,122 @@ impl Contract {
             )
             .into()
         } else {
-            self.process_fin_transfer_to_other_cahin(predecessor_account_id, transfer_message);
+            self.process_fin_transfer_to_other_chain(predecessor_account_id, transfer_message);
             PromiseOrValue::Value(destination_nonce)
         }
     }
 
+    fn fast_fin_transfer(
+        &mut self,
+        sender_id: AccountId,
+        token_id: AccountId,
+        amount: U128,
+        fast_fin_transfer_msg: FastFinTransferMsg,
+    ) -> PromiseOrValue<U128> {
+        let fast_transfer = FastTransfer {
+            token_id: token_id.clone(),
+            recipient: fast_fin_transfer_msg.recipient.clone(),
+            amount: U128(amount.0 + fast_fin_transfer_msg.fee.fee.0),
+            fee: fast_fin_transfer_msg.fee,
+            transfer_id: fast_fin_transfer_msg.transfer_id,
+            msg: fast_fin_transfer_msg.msg,
+        };
+
+        match fast_fin_transfer_msg.recipient {
+            OmniAddress::Near(recipient) => {
+                let storage_deposit_amount = fast_fin_transfer_msg
+                    .storage_deposit_amount
+                    .unwrap_or_default();
+                if storage_deposit_amount > 0 {
+                    self.update_storage_balance(
+                        sender_id.clone(),
+                        NearToken::from_yoctonear(storage_deposit_amount),
+                        NearToken::from_yoctonear(0),
+                    );
+                }
+
+                let deposit_action = StorageDepositAction {
+                    account_id: recipient,
+                    token_id,
+                    storage_deposit_amount: fast_fin_transfer_msg.storage_deposit_amount,
+                };
+                PromiseOrValue::Promise(
+                    Self::check_or_pay_ft_storage(
+                        &deposit_action,
+                        &mut NearToken::from_yoctonear(storage_deposit_amount),
+                    )
+                    .then(
+                        Self::ext(env::current_account_id())
+                            .with_static_gas(VERIFY_PROOF_CALLBACK_GAS)
```

Comment:
The const `VERIFY_PROOF_CALLBACK_GAS` isn't correct

Reply:
@kiseln

Reply:
Fixed 1f38b0469619ad975744e128cc91dc404cde9b6f